router/pkg/config/config.schema.json

```diff
@@ -694,6 +694,13 @@
           "properties": {
             "fields": {
               "$ref": "#/$defs/context_fields"
+            },
+            "ignore_query_params_list": {
```

Comment:
We should skip logging `variables` by default

Reply:
Will add

Reply:
resolved